aws/secrets.mk

```diff
@@ -17,6 +18,14 @@ CMD_SERVICE_SECRETS_DELETE = @ (echo $(foreach item, $(shell aws ssm --profile=$
 		$(shell aws ssm --profile=$(AWS_PROFILE) delete-parameter --name $(item))) > /dev/null ) && \
 		echo "\033[32m[OK]\033[0m /$(ENV)/$(SVC)/* secrets deleted" || echo "\033[31m[ERROR]\033[0m /$(ENV)/$(SVC)/* secrets deletion"
 
+CMD_SERVICE_ALL_SECRET_KEYS = $(foreach item, $(shell aws ssm --profile=$(AWS_PROFILE) get-parameters-by-path \
+		--path "/$(ENV)/$(SVC)" --recursive | $(JQ) -e -r '.Parameters[] | select(.Type == "SecureString") | .Name' ), $(item))
+CMD_SERVICE_SECRETS_PULL = @ (echo $(shell echo "{\"INFO\":\"EMPTY_JSON_CREATED\"}" > $(SERVICE_SECRETS_BACKUP_FILE)) && \
+		$(foreach item, $(CMD_SERVICE_ALL_SECRET_KEYS), \
+		$(shell $(JQ) --arg value "$(shell echo $(item) | sed 's|.*/||')" '.[$$value] = "'$(shell aws ssm --profile=$(AWS_PROFILE) get-parameter --with-decryption --name $(item) --query Parameter.Value)'"' \
+		$(SERVICE_SECRETS_BACKUP_FILE) > tmp.json && mv tmp.json $(SERVICE_SECRETS_BACKUP_FILE))) > /dev/null ) && \
```

Comment:
Can we avoid mv here? Use pipe, maybe something like `-`?

Reply:
@igorkotof also, we should make sure we add this to SLS alias too

Reply:
@AutomationD you know this workaround is necessary measure due to JQ working process, pipe doesn't work here :( . What do you mean under `-` ?